fork.yaml

```diff
@@ -166,7 +168,6 @@ def:
             - "eth/ethconfig/config.go"
         - title: Tx gossip disable option
           globs:
-            - "eth/handler.go"
```

Comment:
I think it spit out the error because of the usage of this diff in two places. We should add a description of the `eth/handler.go` changes in one place, to keep some documentation of the total set of changes.